addons/point_of_sale/models/pos_session.py

```diff
@@ -1716,18 +1716,29 @@ def _loader_params_account_tax(self):
             'search_params': {
                 'domain': [('company_id', '=', self.company_id.id)],
                 'fields': [
-                    'name', 'real_amount', 'price_include', 'include_base_amount', 'is_base_affected',
-                    'amount_type', 'children_tax_ids'
+                    'name', 'price_include', 'include_base_amount', 'is_base_affected',
+                    'amount_type', 'children_tax_ids', 'amount', 'id'
                 ],
             },
         }
 
     def _get_pos_ui_account_tax(self, params):
         taxes = self.env['account.tax'].search_read(**params['search_params'])
-        # TODO: rename amount to real_amount in front end
+
+        # Add the 'sum_repartition_factor' as needed in the compute_all
+        # Note that the factor = factor_percent/100
+        tax_ids = tuple([t['id'] for t in taxes])
+        self.env.cr.execute("""
+            SELECT invoice_tax_id AS tax_id, sum(factor_percent)/100 AS factor_sum
+            FROM account_tax_repartition_line 
+            WHERE invoice_tax_id IN %s AND repartition_type = 'tax' 
+            GROUP BY invoice_tax_id
+        """, [tax_ids])
+        res = self.env.cr.dictfetchall()
```

Comment:
Pretty sure this can be done using the ORM. Any valid reason to straight use SQL query ?

Reply:
You can still use the read_group, but yeah, I prefer using the ORM for trivial and simple request (otherwise we should always use SQL queries) like that unless the performance gain is extremely worth it or the request is quite complex/impossible to do using the ORM.